### PR TITLE
[WIP] Fix summarize.result

### PR DESCRIPTION
**Pull request by jessicaguo**
Function did not summarize SE due to an ordering issue. n and mean were moved to below statname and stat. 
Additional tests were created to ensure the new function works correctly. 

## Review Time Estimate

- [ ] Immediately
- [x] Within one week
- [ ] When possible
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by infotroph** [comment]
Are the changes to `.github/workflows/depends.yml` and `.github/workflows/docker.yml` intentional or a merge mixup?

File: base/utils/tests/testthat/test.utils.R [hunk 1]
```diff
@@ -69,13 +69,30 @@ test_that("summarize.result works appropriately", {
...
+  expect_equal(round(testresult3$stat, 3), c(0.359, 0.177, 0.293, 0.206))
+  expect_equal(round(testresult3$mean, 3), c(1.656, 2.823, 1.707, 2.813))
+  
+  # check that site groups correctly for length(site) > 1
+  testresult4 <- rbind.data.frame(testresult2, transform(testresult2, site_id= 2))
+  testresult5 <- summarize.result(testresult4)
+  expect_true(all(!is.na(testresult5$stat)))
+  expect_equal(nrow(testresult5), 8)
+  
+  })
```
Comment (infotroph) [inline base/utils/tests/testthat/test.utils.R:95]: Thank you for including unit tests! We've been, uh, inconsistent about them recently.

**Comment by dlebauer** [comment]
this is a borked PR; fixing now

**Comment by dlebauer** [comment]
replaced by #2753

**Pull request closed without merging**